force-app/main/service/AddressesServiceImpl.cls

```diff
@@ -127,6 +127,15 @@ public inherited sharing class AddressService {
         return str1.equals(str2);
     }
 
+    public void normalizeLineBreaksFor(List<SObject> records) {
```

Comment:
This logic still exists in `AccountsServiceImpl`, let's try to move that call here.